skyportal/models.py

```diff
@@ -951,6 +959,14 @@ def candidate_is_readable_by(self, user_or_token):
 Candidate.is_readable_by = candidate_is_readable_by
 
 
+User.listings = relationship(
+    'Listing',
+    backref='listings',
```

Comment:
```suggestion
    back_populates='user',
```

Reply:
You only use backref when you don't have a relationship declared on the other model, but in this case you do, so each relationship should use back_populates. Also it should be "user" so that this links up to the user relationship defined on Listing

Reply:
https://docs.sqlalchemy.org/en/14/orm/backref.html

Reply:
Yeah I was very confused about the difference between these... Thanks!